src/compile-middleware.ts

```diff
@@ -66,6 +67,13 @@ function isSuccessful(response: Response) {
   return (statusCode >= 200 && statusCode < 300);
 }
 
+// NOTE: To change the max length of the cache at runtime, just use bracket
+// notation, i.e. `babelCompileCache['max'] = 64 * 1024` for 64KB limit.
+export const babelCompileCache = LRU<string>(<LRU.Options<string>>{
```

Comment:
I think its a side-effect of the typings file having two declarations for babelCompileCache (one takes a `number`, the other takes an `LRU.Options<T>`) and for some reason typescript can't figure out that my object matches the `LRU.Options` type without explicitly typing it out, even when I say `LRU<string>` it says my `{...}` isn't a number and makes me feel sad.